utils.sh

```diff
@@ -290,7 +290,7 @@ function generate_metal3_config {
 }
 
 function image_mirror_config {
-    if [ ! -z "${MIRROR_IMAGES}" ]; then
+    if [ ! -z "${ENABLE_LOCAL_REGISTRY:-${MIRROR_IMAGES:-}}" ]; then
```

Comment:
Is this whole block really required also in case of plain registry only? It seems it contains references to some stuff used only in case of mirroring release image (ie `local-release-mage`)

Reply:
Most probably you are right. For example, we might be using the inner blocks of `if [ ! -s ${MIRROR_LOG_FILE} ]; then` only. But do you think that we should separate the logic for MIRROR_IMAGES and ENABLE_LOCAL_REGISTRY?

Reply:
Yes, they are two different features for two different scenarios. `MIRROR_IMAGES` is required for disconnected _release_ images (which is a mandatory condition for installing ipv6 scenario, but it's not for ipv4).
`ENABLE_LOCAL_REGISTRY` instead is meant to support disconnected _test_ images (which is something always required for all the test configurations we're supporting right now).